me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -50,11 +51,11 @@
       <classpathProvider
           id="me.gladwell.m2e.android.sourcepathProvider"
           class="me.gladwell.eclipse.m2e.inject.GuiceExtensionFactory:me.gladwell.eclipse.m2e.android.JUnitSourcepathProvider" />
-   </extension>
-   <extension
-         point="org.eclipse.m2e.core.mavenProjectChangedListeners">
-      <listener
-            class="me.gladwell.eclipse.m2e.inject.GuiceExtensionFactory:me.gladwell.eclipse.m2e.android.EclipseEventsToEventBusListener">
-      </listener>
-   </extension>
+   </extension>
```

Comment:
You added CRLFs to the last part of the file. Download [this](https://raw.githubusercontent.com/WonderCsabo/m2e-android/61a737c11af6d3a3cd2da001b3d2ac22da405766/me.gladwell.eclipse.m2e.android/plugin.xml) and see for yourself.